src/test/kotlin/io/github/enyason/base/RetrofitFactoryTest.kt

```diff
@@ -2,39 +2,37 @@ package io.github.enyason.base
 
 import okhttp3.OkHttpClient
 import okhttp3.logging.HttpLoggingInterceptor
-import org.junit.jupiter.api.MethodOrderer
-import org.junit.jupiter.api.Order
-import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.TestMethodOrder
 import org.junit.jupiter.api.assertThrows
 import retrofit2.converter.gson.GsonConverterFactory
-import kotlin.test.assertEquals
+import kotlin.test.AfterTest
+import kotlin.test.Test
 import kotlin.test.assertNotNull
 import kotlin.test.assertTrue
 
-@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
 class RetrofitFactoryTest {
 
+    @AfterTest
+    fun tearDown() {
+        RetrofitFactory.reset()
+    }
```

Comment:
Due to the fact that RetrofitFactory is a singleton, the tests in this class were affecting the tests in PredictionsApiTest. Hence the need for making Retrofit nullable and creating the reset function